### DISCUSSION TITLE
List (more) related projects

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/tosdr.org#207] [Ideas]
would be helpful if we link to projects like toschecker.com, privacygrade.org, tosback.org and https://factoryjoe.pbworks.com/w/browse/#view=ViewFolder&param=Legalese

**Comment by michielbdejong** [comment]
* user data manifesto

**Comment by michielbdejong** [comment]
* https://robb.weblaws.org/2015/10/17/os-x-el-capitan-license-in-plain-english/

**Comment by evilcel3ri** [comment]
I agree !
